tests/manage/pv_services/test_ceph_daemon_kill_during_resource_creation.py

```diff
@@ -0,0 +1,392 @@
+            wait_each=False
+        )
+
+        if operation_to_disrupt == 'create_pvc':
+            # Ensure PVCs are being created before deleting the resource
+            ret = self.verify_resource_creation(
+                get_all_pvcs, initial_num_of_pvc, namespace
+            )
+            assert ret, "Wait timeout: PVCs are not being created."
+            log.info("PVCs creation has started.")
+            disruption.kill_daemon()
+
+        pvc_objs = bulk_pvc_create.result()
+
+        # Confirm that PVCs are Bound
+        for pvc_obj in pvc_objs:
+            helpers.wait_for_resource_state(
+                resource=pvc_obj, state=constants.STATUS_BOUND, timeout=120
+            )
+            pvc_obj.reload()
+        log.info("Verified: PVCs are Bound.")
+
+        # Start creating pods
+        bulk_pod_create = executor.submit(
+            self.pods_creation, pvc_objs, pod_factory, interface
+        )
+
+        if operation_to_disrupt == 'create_pod':
+            # Ensure that pods are being created before deleting the resource
+            ret = self.verify_resource_creation(
+                pod.get_all_pods, initial_num_of_pods, namespace
+            )
+            assert ret, "Wait timeout: Pods are not being created."
+            log.info(f"Pods creation has started.")
+            disruption.kill_daemon()
+
+        pod_objs = bulk_pod_create.result()
+
+        # Verify pods are Running
+        for pod_obj in pod_objs:
+            helpers.wait_for_resource_state(
+                resource=pod_obj, state=constants.STATUS_RUNNING
+            )
+            pod_obj.reload()
+        log.info("Verified: All pods are Running.")
+
+        # Do setup on pods for running IO
+        log.info("Setting up pods for running IO.")
+        for pod_obj in pod_objs:
+            pvc_info = pod_obj.pvc.get()
+            if pvc_info['spec']['volumeMode'] == 'Block':
+                storage_type = 'block'
+            else:
+                storage_type = 'fs'
+            executor.submit(pod_obj.workload_setup, storage_type=storage_type)
+
+        # Wait for setup on pods to complete
+        for pod_obj in pod_objs:
+            for sample in TimeoutSampler(
+                180, 2, getattr, pod_obj, 'wl_setup_done'
+            ):
+                if sample:
+                    log.info(
+                        f"Setup for running IO is completed on pod "
+                        f"{pod_obj.name}."
+                    )
+                    break
+        log.info("Setup for running IO is completed on all pods.")
+
+        # Start IO on each pod
+        for pod_obj in pod_objs:
+            pvc_info = pod_obj.pvc.get()
+            if pvc_info['spec']['volumeMode'] == 'Block':
+                storage_type = 'block'
+            else:
+                storage_type = 'fs'
+            pod_obj.run_io(
+                storage_type=storage_type, size='1G', runtime=10,
```

Comment:
1G and runtime are too small/short.
I think that by the time you will kill the daemon, there will be pods already done IOing which will create a non-deterministic test case

Reply:
I think that we are ok as long as there are *some* pods still IOing.

Reply:
There is chance that IO will be finished on some pods. Some will have IO running. As a better approach to have all pods IOing during the failure, I will increase the runtime.

Reply:
Increased size and runtime paramaters